asset-tracking-history-saver/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "asset-tracking-history-saver",
+  "version": "1.0.0",
+  "description": "CLI tool that saves enhanced location updates history from a channel to a file",
+  "main": "index.ts",
+  "scripts": {
+    "start": "node ./lib/index.js",
+    "build": "tsc -p ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
We should use the apache 2 license unless there's a good reason not to do so here

Reply:
I've changed the license https://github.com/ably/ably-asset-tracking-common/pull/43/commits/03df57398a239c933decd31f6fe4f879a9a18dc3, however, all the other scripts in this repository have the `ISC` license as well :thinking: Should we create an issue for changing their licenses to `Apache-2.0`?